java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -1236,24 +1080,14 @@ public void shouldInitialiseRootPartitionCorrectlyForStringKey() throws StateSto
         // Given
         Field field = new Field("key", new StringType());
         Schema schema = Schema.builder().rowKeyFields(field).build();
-        RangeFactory rangeFactory = new RangeFactory(schema);
         StateStore dynamoDBStateStore = getStateStore(schema);
-
         // When
         List<Partition> partitions = dynamoDBStateStore.getAllPartitions();
-
         // Then
         assertThat(partitions).hasSize(1);
```

Comment:
We could remove this assertion, as containsExactly already asserts this with better output.